editor.planx.uk/src/pages/FlowEditor/lib/store/settings.ts

```diff
@@ -51,6 +54,20 @@ export const settingsStore: StateCreator<
     return Boolean(result?.id);
   },
 
+  flowDescription: "",
+
+  setFlowDescription: (flowDescription: string) => set({ flowDescription }),
+
+  updateFlowDescription: async (newDescription: string) => {
+    const { id, $client } = get();
+    const result = await $client.flow.setDescription({
+      flow: { id },
+      description: newDescription,
+    });
+    set({ flowDescription: newDescription });
```

Comment:
Despite this change, I'm not seeing the value updated when I toggle pages - 


https://github.com/user-attachments/assets/986b3c7c-a1e6-439d-9795-606d9098a2c1

Reply:
This is happening because Apollo is overwriting Zustand's state on route change. 

I think this is a much broader problem we need to solve across the app rather than just patching it here there and everywhere, but in the meantime we shouldn't introduce buggy behaviour.

Reply:
Ah yes, I forgot this from the team settings stuff, I'll put up a fix now! thanks for catching this